sdk/monitor/monitor-query/README.md

```diff
@@ -394,6 +395,99 @@ async function processTables(tablesFromResult: LogsTable[]) {
 
 A full sample can be found [here](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-query/samples/v1/typescript/src/logsQueryBatch.ts).
 
+### Resource Logs Query
```

Comment:
I recommend deleting this new section and reworking the existing "Logs query" section. See the .NET library for an example: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/monitor/Azure.Monitor.Query#logs-query